src/main/java/Bond/Bond.java

```diff
@@ -0,0 +1,325 @@
+        String[] splitInput = input.split("/to", 2);
+        String eventDescription = splitInput[0].substring(EVENT.length() + 1, splitInput[0].indexOf("/from")).trim();
+        String fromDescription = splitInput[0].substring(splitInput[0].indexOf("/from") + "/from".length()).trim();
+        String toDescription = splitInput[1].trim();
+        if (eventDescription.isEmpty() || fromDescription.isEmpty() || toDescription.isEmpty()) {
+            throw new IllegalArgumentException();
+        }
+        return new Event(eventDescription, fromDescription, toDescription);
+    }
+
+    public static void deleteTask(String input) {
+        try {
+            String[] splitInput = input.split(" ", 2);
+            boolean isValidInput = splitInput.length == 2 && !splitInput[1].trim().isEmpty();
+            if (!isValidInput) {
+                throw new IllegalArgumentException();
+            }
+            tasks.remove(Integer.parseInt(splitInput[1]) - 1);
+            System.out.println(TAB + String.format(TASK_DELETED, splitInput[1]));
+        } catch (IllegalArgumentException e) {
+            System.out.println(TAB + DELETE_USAGE);
+        } catch (IndexOutOfBoundsException e) {
+            System.out.println(TAB + INVALID_TASK_NUMBER);
+        }
+    }
+
+    public static void executeCommand(String input) {
+        try {
+            String command = input.split(" ", 2)[0];
+            switch (command) {
+            case LIST:
+                printList();
+                break;
+            case MARK:
+            case UNMARK:
+                markTask(input);
+                break;
+            case TODO:
+                addTodo(input);
+                break;
+            case DEADLINE:
+                addDeadline(input);
+                break;
+            case EVENT:
+                addEvent(input);
+                break;
+            case DELETE:
+                deleteTask(input);
+                break;
+            default:
+                throw new Exceptions.IllegalArgumentException();
+            }
+        } catch (IllegalArgumentException e) {
+            System.out.println(TAB + INVALID_COMMAND);
+        } finally {
+            System.out.print(COMMAND);
+        }
+    }
+
+    public static void createFile(String filePath) {
+        File taskFile = new File(filePath);
+        File parentDirectory = taskFile.getParentFile();
+
+        if (parentDirectory != null && !parentDirectory.exists()) {
+            boolean dirsCreated = parentDirectory.mkdirs();
+            if (dirsCreated) {
+                System.out.println("Directories created: " + parentDirectory.getAbsolutePath());
+            } else {
+                System.out.println("Failed to create directories!");
+            }
+        }
+        if (!taskFile.exists()) {
+            try {
+                if (taskFile.createNewFile()) {
+                    System.out.println("File created: " + taskFile.getAbsolutePath());
```

Comment:
This is around 3~ levels of indentation, consider refactoring the inner levels into their own functions